catchpoint/Makefile

```diff
@@ -0,0 +1,71 @@
+VM_NAME := catchpoint
+K8S_NAMESPACE := default
+CONFIG_FILE := jinja/variables/cloud-init.yaml
+TEMPLATE_DIR := jinja/templates
+OUTPUT_DIR := generated_configs
+GRAFANA_FLOW_VALUES := $(OUTPUT_DIR)/grafana-flow-k8s-template.yaml
+
+.PHONY: run-catchpoint-multipass
+run-catchpoint-multipass: render-config
```

Comment:
This runs render-config as part of the setup process. Not sure that's intended.

Reply:
yeah, it's good. I forgot one two many times to render after making jinja template changes. I think this removes the need of running the step 3 `make render-config` too.